tests/testthat/test-run_stress_test.R

```diff
@@ -0,0 +1,23 @@
+test_that("is sensitive to `asset_type`, and defaults to bonds", {
```

Comment:
pls rm default to bonds, s.a.

Reply:
RE:

> I would not see a benefit in returning anything other than NULL/invisible(asset_type) here.

Here I'm inspired by https://design.tidyverse.org/out-invisible.html

> If a function is called primarily for its side-effects, it should invisibly return a useful output. If there’s no obvious output, return the first argument. This makes it possible to use the function with in a pipeline.

``` r
# Good
f <- function(x) {
  writeLines("Nothing is always something", tempfile())
  invisible(x)
}

1 |> f() |> sum(2)
#> [1] 3

# Bad
f <- function(x) {
  writeLines("Nothing is always something", tempfile())
}

1 |> f() |> sum(2)
#> [1] 2
```

Reply:
here again I would want to be pragmatic and consider the purpose of the function. The sole purpose of the function is its sideeffect, i.e. wrapping around a pipeline of data import, some calculation, and export. The nature of the function make it not useful to potentially include it in a pipeline, thus I would opt for a NULL/invisible() return here.